shared-bindings/microcontroller/Processor.c

```diff
@@ -30,9 +30,20 @@
 #include <math.h>
 #include <stdint.h>
 
-#include "py/objproperty.h"
+// #include "py/objproperty.h"
+
+// #include "py/runtime.h"
```

Comment:
This looks like some kind of leftover. Just remove?

Reply:
Deleted